### ISSUE TITLE
QImg events types

### ISSUE THREAD

**Issue by KonstantinSerebryakov** [quasarframework/quasar#16545]
### Description

documentation says @error take src parameter:
```
src: Error
Description
JS Error object
```
while at runtime it actually pass Event with type 'error' object:

template:
```
  <q-img
    @error="handleError"
    src="someinvalidsrc.png"
  ></q-img>
````
script:
````
function handleError(e) {
    console.log(e instanceof Event);
}
````

### Documentation Section URL

https://quasar.dev/vue-components/img#qimg-api

### Flavour

None

### Areas

TypeScript Support

**Comment by pdanpdan** [comment]
You can make a PR for this - the description is good, only the parameter name is strange

**Comment by rstoenescu** [comment]
Fix will be available in Quasar v2.13.1

**Comment by yusufkandemir** [comment]
@rstoenescu I think the main point here is the payload being an instance of `Event`, not `Error`.
https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/error_event#event_type

**Comment by pdanpdan** [comment]
Unfortunately, checking the code, it looks like it can be Event (from `img` `onerror`) and Error (from `img.decode`)

**Comment by rstoenescu** [comment]
@yusufkandemir 
The native "error" event emitted by the IMG tag has an Event as param. We are simply passing it over to the user.
Some info - https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/error_event

**Comment by yusufkandemir** [comment]
@rstoenescu yes, I am aware of that, that's the point. We are emitting `Event` but we are using `Error` as the type, which is incorrect.

**Comment by rstoenescu** [comment]
@yusufkandemir thanks for pointing this out.

Final fix will be available on Quasar v2.13.2
